game-core/src/main/java/org/triplea/game/server/HeadlessLaunchAction.java

```diff
@@ -31,8 +31,8 @@ public void handleGameInterruption(
       // then crashing out, then launching, etc.
       serverModel.setAllPlayersToNullNodes();
       final File f1 = getAutoSaveFileUtils().getHeadlessAutoSaveFile();
-      if (!f1.exists() || !gameSelectorModel.load(f1)) {
-        gameSelectorModel.resetGameDataToNull();
+      if (!f1.exists()) {
+        gameSelectorModel.load(f1);
```

Comment:
I believe a small bug sneaked in here:
Load should only get invoked when the file does actually exist, not the other way round.
Previously in the if statement this was the case because the second or case was only ever executed if `!f1.exists()` was false i.e. `f1.exists()` was true.

Reply:
You're right, that is a good catch.
Fixed in: 7846a1bfa